js/bybit.js

```diff
@@ -1162,19 +1162,18 @@ module.exports = class bybit extends Exchange {
     }
 
     async fetchDerivativesMarkets (params) {
-        params['limit'] = 1000; // minimize number of requests
+        params['limit'] = 1000; // maximize number of requests
```

Comment:
@sc0Vu  I thought we wanted to minimize the number of requests made by ccxt 😅

Reply:
@carlosmiei Ah, you're right! I thought "maximize limit number".